apps/backend/src/routes/onboard/verification/+page.svelte

```diff
@@ -0,0 +1,72 @@
+<!-- JavaScript code -->
+<script lang="ts">
+  import { firestore } from '$lib/firebase';
+  import type { FirebaseError } from 'firebase/app';
+  import { authStore, authState } from '$lib/stores';
+  import { doc, getDoc } from 'firebase/firestore';
+  import { verifyEmail, signOut } from '$lib/firebase/auth';
+
+  let apiError: FirebaseError | null;
+</script>
+
+<svelte:head>
+  <title>Email Verification</title>
+</svelte:head>
+
+<div class="mt-32 flex flex-row items-center justify-center">
+  <!-- Email Verification - Card -->
+  <div class="w-full max-w-2xl p-10 border border-secondary rounded-md shadow-lg">
+    <!-- Email Verification - Header -->
+    <h1 class="text-3xl font-semibold text-center mb-5">Please verify your email!</h1>
+
+    <!-- Email Verification - Content -->
+    <div class="text-center">
+      <p>You're almost there! We have sent an email to</p>
+
+      <!-- User Details -->
+      {#if $authStore?.email}
+        <div class="text-info">
+          {#if $authStore?.email}
+            {$authStore?.email.slice(0, 1) +
+              '*****' +
+              $authStore?.email.slice($authStore?.email.indexOf('@'))}
+          {/if}
+        </div>
+      {:else}
+        <span class="text-error">Loading user data...</span>
+      {/if}
```

Comment:
Why double `{#if $authStore?.email}`? Makes no difference to me

Reply:
Also, slicing the data on the client side here really doesn't do anything. Anyone can just open devtools and pull the email address out

Reply:
I think i didnt see that i actually put it twice, my bad on this as well.

Simply removed the redundancy:
```
<!-- User Details -->
      {#if $authStore?.email}
        <div class="text-info">
          {$authStore?.email.slice(0, 1) +
            '*****' +
            $authStore?.email.slice($authStore?.email.indexOf('@'))}
        </div>
      {:else}
        <span class="text-error">Loading user data...</span>
      {/if}
```

Permission to keep the slicing though, I saw it on multiple websites, yes even though can just open devtools I thought it would be a good impression if I implemented it :)